content/_data/changelogs/lts.yml

```diff
@@ -3913,6 +3913,590 @@
+        title: security advisory
+    authors:
+      - daniel-beck
+      - wadeck
+  - type: major bug
+    category: regression
+    pull: 4912
+    issue: 63458
+    authors:
+    - jglick
+    message: |-
+      Stop pre-formatting agent logs to prevent deadlocks (regression in 2.231).
+  - type: major bug
+    category: regression
+    pull: 4897
+    issue: 63274
+    authors:
+    - Wadeck
+    message: |-
+      Fix button that copies API token to clipboard (regression in 2.238).
+  - type: major bug
+    category: regression
+    pull: 4870
+    issue: 63180
+    authors:
+    - fqueiruga
+    message: |-
+      Restore wrapping tabs into multiple lines instead of overflowing (regression in 2.248).
+  - type: bug
+    category: bug
+    references:
+      - url: https://github.com/jenkinsci/dark-theme-plugin/issues/126
+        title: Fixes bread crumbs flash in Dark Theme
+    issue: 63285
+    pull: 4867
+    authors:
+    - fqueiruga
+    message: |-
+      Normalize widget colors to be consistent with the new color palette.
+  - type: bug
+    category: regression
+    pull: 4892
+    issue: 63276
+    authors:
+    - fqueiruga
+    message: |-
+      Empty installed plugins table text is readable again (regression in 2.249).
+  - type: bug
+    category: bug
+    pull: 4893
+    issue: 63232
+    authors:
+    - Wadeck
+    message: |-
+      Show build time data in the Build Time Trend Page (regression in 2.245).
+  - type: bug
+    category: bug
+    pull: 4863
+    issue: 63166
+    authors:
+    - takashiharano
+    message: |-
+      Replace text references to slave with agent in Japanese documentation and messages.
+  - type: bug
+    category: regression
+    pull: 4901
+    issue: 63342
+    authors:
+    - francoiscampbell
+    message: |-
+      Fix backspace key sometimes did not delete text from the Script Console on a Mac (regression in 2.248).
+  - type: bug
+    category: regression
+    pull: 4896
+    issue: 63308
+    authors:
+    - timja
+    message: |-
+      Fix regular expression validator UI location (regression in 2.244).
+  - type: rfe
+    category: rfe
+    pull: 4850
+    issue: 61687
+    authors:
+    - gmshake
+    message: |-
+      Prevent concurrent build deletion.
+
+  lts_changes: # compared to lts_predecessor 2.235.5 (selected by personal review)
+
+  # Major user experience improvements
+  - type: major rfe
+    category: major rfe
+    pull: 4752
+    authors:
+    - 95jonpet
+    - timja
+    - fqueiruga
+    references:
+      - issue: 60924
+      - pull: 4752
+      - issue: 62515
+      - pull: 4763
+      - pull: 4772
+      - pull: 4814
+      - pull: 4842
+      - pull: 4843
+      - url: https://github.com/jenkinsci/dark-theme
+        title: Dark Theme repository
+      - url: https://cd.foundation/blog/2020/08/04/introducing-the-jenkins-dark-theme/
+        title: Introducing the Jenkins Dark Theme
+    message: |-
+      Release 'alpha' dark theme.
```

Comment:
Would rephrase as 'Add support for theming' or similar, and then link to Dark Theme. The core change is the support, people still need to install this.